project/Settings.scala

```diff
@@ -9,6 +9,7 @@ object Settings {
     val cats              = "2.1.1"
     val catsEffect        = "2.1.3"
     val circe             = "0.13.0"
+    val contextApplied    = "0.1.3"
```

Comment:
What's this? I don't see it used in `build,sbt`

Reply:
Briefly tried to used `context-applied` and forgot to remove the version declaration from here.

BTW, I rolled back on this because it caused linking errors on `SubscriptionRenderMod` regarding the `Reusability` type class.